integration test on managed deployment causes random build failure

https://github.com/apache/incubator-openwhisk-wskdeploy/tree/master/tests/src/integration/managed-deployment has many different test cases in one single integration test. It causes random build failures, please break it down into multiple different tests to avoid such build failures. I will add more details as I find them with the exact error message.

```
--- FAIL: TestManagedDeployment (5.11s)
	assertions.go:256: 
			Error Trace:	managed-deployment_test.go:37
			Error:      	Not equal: 
			            	expected: <nil>(<nil>)
			            	actual  : *wskderrors.CommandError(&wskderrors.CommandError{WskDeployBaseErr:wskderrors.WskDeployBaseErr{ErrorType:"ERROR_COMMAND_FAILED", FileName:"wskdeploy.go", LineNum:128, Message:"Command: [wskdeploy]: Error: Deployment did not complete sucessfully. Run `wskdeploy undeploy` to remove partially deployed assets.\nError: The requested resource does not exist. (code d35c29029f88325a5c103e8763f36181)", MessageFormat:"%s: [%s]: %s"}, Command:"wskdeploy"})
			Test:       	TestManagedDeployment
			Messages:   	Failed to deploy based on the manifest and deployment files.
```

--- a/tests/src/integration/managed-deployment/07-manifest-with-dependency.yaml
+++ b/tests/src/integration/managed-deployment/07-manifest-with-dependency.yaml
@@ -2,26 +2,24 @@
 # license agreements; and to You under the Apache License, Version 2.0.
 
 project:
-    name: MyManagedProjectWithDependency
+    name: MyManagedProjectWithMultipleDependency
     packages:
         Extension2:
             dependencies:
-                helloworlds:
-                    location: github.com/apache/incubator-openwhisk-test/packages/helloworlds
-                custom-hellowhisk:
+                hellowhisk-with-managed-project:
                     location: github.com/apache/incubator-openwhisk-test/packages/hellowhisk
-                custom-helloworlds:
+                helloworlds-with-managed-project:
                     location: github.com/apache/incubator-openwhisk-test/packages/helloworlds
             sequences:
                 helloworld-sequence:
-                   actions: custom-helloworlds/hello-js, custom-hellowhisk/greeting, helloworlds/hello-js
+                   actions: helloworlds-with-managed-project/hello-js, hellowhisk-with-managed-project/greeting
             triggers:
-                trigger1:
-                trigger2:
+                trigger1InMultipleDep:
+                trigger2InMultipleDep:
             rules:
                 rule1:
-                    trigger: trigger1
+                    trigger: trigger1InMultipleDep
                     action: helloworld-sequence
                 rule2:
-                    trigger: trigger2
-                    action: custom-helloworlds/helloworld-js
+                    trigger: trigger2InMultipleDep
+                    action: helloworlds-with-managed-project/helloworld-js

--- a/tests/src/integration/managed-deployment/managed-deployment_test.go
+++ b/tests/src/integration/managed-deployment/managed-deployment_test.go
@@ -56,8 +56,13 @@ func TestManagedDeployment(t *testing.T) {
 	_, err = wskdeploy.ManagedDeployment(manifestPath, deploymentPath)
 	assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
 
-	manifestPath = os.Getenv("GOPATH") + PATH + "05-manifest-headless.yaml"
-	_, err = wskdeploy.HeadlessManagedDeployment(manifestPath, deploymentPath, "Headless Managed")
+}
+
+func TestHeadlessManagedDeployment(t *testing.T) {
+	manifestPath := os.Getenv("GOPATH") + PATH + "05-manifest-headless.yaml"
+	deploymentPath := ""
+	wskdeploy := common.NewWskdeploy()
+	_, err := wskdeploy.HeadlessManagedDeployment(manifestPath, deploymentPath, "HeadlessManaged")
 	assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
 	_, err = wskdeploy.Undeploy(manifestPath, deploymentPath)
 	assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
@@ -72,16 +77,24 @@ func TestManagedDeploymentWithDependency(t *testing.T) {
 	assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
 	_, err = wskdeploy.Undeploy(manifestPath, deploymentPath)
 	assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
+}
 
-	manifestPath = os.Getenv("GOPATH") + PATH + "07-manifest-with-dependency.yaml"
-	_, err = wskdeploy.ManagedDeployment(manifestPath, deploymentPath)
+func TestManagedDeploymentWithMultipleDependency(t *testing.T) {
+	manifestPath := os.Getenv("GOPATH") + PATH + "07-manifest-with-dependency.yaml"
+	deploymentPath := ""
+	wskdeploy := common.NewWskdeploy()
+	_, err := wskdeploy.ManagedDeployment(manifestPath, deploymentPath)
 	assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
 	_, err = wskdeploy.Undeploy(manifestPath, deploymentPath)
 	assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
-
-	//manifestPath = os.Getenv("GOPATH") + PATH + "08-manifest-with-dependencies-on-whisk-system.yaml"
-	//_, err = wskdeploy.ManagedDeployment(manifestPath, deploymentPath)
-	//assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
-	//_, err = wskdeploy.Undeploy(manifestPath, deploymentPath)
-	//assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
 }
+
+//func TestManagedDeploymentWithWhiskSystem(t *testing.T) {
+//manifestPath := os.Getenv("GOPATH") + PATH + "08-manifest-with-dependencies-on-whisk-system.yaml"
+//deploymentPath := ""
+//wskdeploy := common.NewWskdeploy()
+//_, err := wskdeploy.ManagedDeployment(manifestPath, deploymentPath)
+//assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
+//_, err = wskdeploy.Undeploy(manifestPath, deploymentPath)
+//assert.Equal(t, nil, err, "Failed to deploy based on the manifest and deployment files.")
+//}

--- a/wski18n/i18n_resources.go
+++ b/wski18n/i18n_resources.go
@@ -329,14 +329,14 @@ func AssetNames() []string {
 
 // _bindata is a table, holding each asset generator, mapped to its name.
 var _bindata = map[string]func() (*asset, error){
-	"wski18n/resources/de_DE.all.json": wski18nResourcesDe_deAllJson,
-	"wski18n/resources/en_US.all.json": wski18nResourcesEn_usAllJson,
-	"wski18n/resources/es_ES.all.json": wski18nResourcesEs_esAllJson,
-	"wski18n/resources/fr_FR.all.json": wski18nResourcesFr_frAllJson,
-	"wski18n/resources/it_IT.all.json": wski18nResourcesIt_itAllJson,
-	"wski18n/resources/ja_JA.all.json": wski18nResourcesJa_jaAllJson,
-	"wski18n/resources/ko_KR.all.json": wski18nResourcesKo_krAllJson,
-	"wski18n/resources/pt_BR.all.json": wski18nResourcesPt_brAllJson,
+	"wski18n/resources/de_DE.all.json":   wski18nResourcesDe_deAllJson,
+	"wski18n/resources/en_US.all.json":   wski18nResourcesEn_usAllJson,
+	"wski18n/resources/es_ES.all.json":   wski18nResourcesEs_esAllJson,
+	"wski18n/resources/fr_FR.all.json":   wski18nResourcesFr_frAllJson,
+	"wski18n/resources/it_IT.all.json":   wski18nResourcesIt_itAllJson,
+	"wski18n/resources/ja_JA.all.json":   wski18nResourcesJa_jaAllJson,
+	"wski18n/resources/ko_KR.all.json":   wski18nResourcesKo_krAllJson,
+	"wski18n/resources/pt_BR.all.json":   wski18nResourcesPt_brAllJson,
 	"wski18n/resources/zh_Hans.all.json": wski18nResourcesZh_hansAllJson,
 	"wski18n/resources/zh_Hant.all.json": wski18nResourcesZh_hantAllJson,
 }
@@ -380,17 +380,18 @@ type bintree struct {
 	Func     func() (*asset, error)
 	Children map[string]*bintree
 }
+
 var _bintree = &bintree{nil, map[string]*bintree{
 	"wski18n": &bintree{nil, map[string]*bintree{
 		"resources": &bintree{nil, map[string]*bintree{
-			"de_DE.all.json": &bintree{wski18nResourcesDe_deAllJson, map[string]*bintree{}},
-			"en_US.all.json": &bintree{wski18nResourcesEn_usAllJson, map[string]*bintree{}},
-			"es_ES.all.json": &bintree{wski18nResourcesEs_esAllJson, map[string]*bintree{}},
-			"fr_FR.all.json": &bintree{wski18nResourcesFr_frAllJson, map[string]*bintree{}},
-			"it_IT.all.json": &bintree{wski18nResourcesIt_itAllJson, map[string]*bintree{}},
-			"ja_JA.all.json": &bintree{wski18nResourcesJa_jaAllJson, map[string]*bintree{}},
-			"ko_KR.all.json": &bintree{wski18nResourcesKo_krAllJson, map[string]*bintree{}},
-			"pt_BR.all.json": &bintree{wski18nResourcesPt_brAllJson, map[string]*bintree{}},
+			"de_DE.all.json":   &bintree{wski18nResourcesDe_deAllJson, map[string]*bintree{}},
+			"en_US.all.json":   &bintree{wski18nResourcesEn_usAllJson, map[string]*bintree{}},
+			"es_ES.all.json":   &bintree{wski18nResourcesEs_esAllJson, map[string]*bintree{}},
+			"fr_FR.all.json":   &bintree{wski18nResourcesFr_frAllJson, map[string]*bintree{}},
+			"it_IT.all.json":   &bintree{wski18nResourcesIt_itAllJson, map[string]*bintree{}},
+			"ja_JA.all.json":   &bintree{wski18nResourcesJa_jaAllJson, map[string]*bintree{}},
+			"ko_KR.all.json":   &bintree{wski18nResourcesKo_krAllJson, map[string]*bintree{}},
+			"pt_BR.all.json":   &bintree{wski18nResourcesPt_brAllJson, map[string]*bintree{}},
 			"zh_Hans.all.json": &bintree{wski18nResourcesZh_hansAllJson, map[string]*bintree{}},
 			"zh_Hant.all.json": &bintree{wski18nResourcesZh_hantAllJson, map[string]*bintree{}},
 		}},
@@ -443,4 +444,3 @@ func _filePath(dir, name string) string {
 	cannonicalName := strings.Replace(name, "\\", "/", -1)
 	return filepath.Join(append([]string{dir}, strings.Split(cannonicalName, "/")...)...)
 }
-

